lumberjack_test.go

```diff
@@ -408,10 +409,6 @@ func TestMaxAge(t *testing.T) {
 	equals(len(b2), n, t)
 	existsWithContent(backupFile(dir), b, t)
 
-	// we need to wait a little bit since the files get deleted on a different
-	// goroutine.
-	<-time.After(10 * time.Millisecond)
```

Comment:
This was waiting for something that shouldn't happen.  Now we'll get a deadlock in the test is there's an extra remove because nobody reads the notifyRemove chan.